lib/Mitigation.pm

```diff
@@ -381,24 +381,30 @@ sub remove_parameter {
 
 sub ssh_vm_cmd {
     my ($cmd, $qa_password, $vm_ip_addr) = @_;
-    my $ret = script_run("sshpass -p ${qa_password} ssh -o StrictHostKeyChecking=no -qy root\@${vm_ip_addr} \"$cmd\"");
+    my $ret = script_run("sshpass -p ${qa_password} ssh -o StrictHostKeyChecking=no -o ConnectTimeout=5 -qy root\@${vm_ip_addr} \"$cmd\"", timeout => 5);
```

Comment:
There are two timeout setting,  if it is duplicate?

Reply:
one is for ssh and other is the openqa command. You are right ,maybe one is enough.

Reply:
one is for ssh and other is the openqa command. You are right ,maybe one is enough.